Tests/Regression/Agent/CommandLine_Common.robot

```diff
@@ -90,5 +90,77 @@ Query Result DB
 	Disconnect From Database
 	RETURN 	${result}
 
+Get Modules From Program .py File That Are Not BuildIn
+	[Tags]	windows-latest	ubuntu-latest	macos-latest	Issue #123
+	[Arguments]		${file_path}	${last_line_number_of_notbuildin}
+	${manager_content}	Get File	${file_path}
+	${manager_content_lines}	Split String	${manager_content}	separator=\n
+	
+	${j}	Set Variable	${0}
+	${j_old}	Set Variable	${0}
+	FOR  ${i}  IN RANGE  0  ${last_line_number_of_notbuildin}
+		${j_old}	Set Variable	${j}
+		${j}	Get Index From List	${manager_content_lines}	start=${j + 1}		value= 
+	END
```

Comment:
Also not sure what you're trying to do with this loop?

Reply:
This loop will set line numbers  that have no build-in variables between them. Again I'm not a fan of this solution but I couldn't find better one at that moment.

Reply:
The problem is you could have 4 builtin, 5 dependency imports, another  few builtin's and then some more dependency imports, repeat, this is because the linter wants them sorted in a specific way and will error if you try to group them in a non standard way

OK The better approach would be:
- define list of known builtin modules
- for loop over every line of the file from first line
  - if import (import or from like you already have) append to imports list
  - if line starts with class break loop (no more imports, it's python file syntax)
- remove the builtin modules from the imports list, return the remaining list